File: 1712.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        int infraCost = sc.nextInt();
        int productionCost = sc.nextInt();
        int productPrice = sc.nextInt();
        int breakEvenQtt = 0;
        
        if(productPrice - productionCost <= 0)
            breakEvenQtt = -1;
        else
            breakEvenQtt = infraCost/(productPrice - productionCost) + 1;
        
        System.out.println(breakEvenQtt);
    }
}

File: 1712.c
#include<stdio.h>

int main(){
    int infraCost = 0;
    int productionCost = 0;
    int productPrice = 0;
    int breakEvenQtt = 0;
    
    scanf("%d %d %d", &infraCost, &productionCost, &productPrice);
    
    if(productPrice - productionCost <= 0)
        breakEvenQtt = -1;
    else
        breakEvenQtt = infraCost/(productPrice - productionCost) + 1;
        
    printf("%d", breakEvenQtt);
    return 0;
}
